.github/workflows/lint.yml

```diff
@@ -40,6 +40,17 @@ jobs:
           (cd src/test && cargo fmt -- --check)
           (cd src/support/logger/rust_bindings && cargo fmt -- --check)
 
+  lint-cargo-lock:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cargo update check
+        run: |
+          (cd src/main && cargo update --package shadow-rs)
```

Comment:
Could probably use a comment explaining what this is trying to catch (and maybe what to do if this test fails).

I think it's ensuring that after we bump Shadow's version we run these commands to sync the version in the Cargo.lock files?

Reply:
I added a comment, and also changed the method that it uses to check since I found a better way. I also updated the "maintainer_playbook.md" doc to explain how to update the lock file versions when updating the Shadow version.

I'm remarking you as reviewer.